src/debian/12/helix/amd64/Dockerfile

```diff
@@ -34,9 +34,9 @@ RUN apt-get update && \
 ENV LANG=en_US.utf8
 
 RUN ln -sf /usr/bin/python3 /usr/bin/python && \
-    python -m pip install virtualenv==20.17.1 && \
-    python -m pip download --no-deps helix-scripts --index-url https://dnceng.pkgs.visualstudio.com/public/_packaging/helix-client-prod/pypi/simple && \
-    python -m pip install ./helix_scripts-*-py3-none-any.whl
+    python3 -m pip install virtualenv==20.17.1 --break-system-packages && \
+    python3 -m pip download --no-deps helix-scripts --index-url https://dnceng.pkgs.visualstudio.com/public/_packaging/helix-client-prod/pypi/simple && \
+    python3 -m pip install ./helix_scripts-*-py3-none-any.whl --break-system-package
```

Comment:
isn't this `--break-system-packages`

Reply:
probably cut & paste error.
But it seems to work 
```
root@c76f85a8e096:/#  python3   -m pip install virtualenv==20.17.1 --break-system-package
Requirement already satisfied: virtualenv==20.17.1 in /usr/local/lib/python3.11/dist-packages (20.17.1)
Requirement already satisfied: distlib<1,>=0.3.6 in /usr/local/lib/python3.11/dist-packages (from virtualenv==20.17.1) (0.3.6)
Requirement already satisfied: filelock<4,>=3.4.1 in /usr/local/lib/python3.11/dist-packages (from virtualenv==20.17.1) (3.9.0)
Requirement already satisfied: platformdirs<3,>=2.4 in /usr/local/lib/python3.11/dist-packages (from virtualenv==20.17.1) (2.6.2)
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
```

Reply:
huh ok then

Reply:
updated anyway for consistency.